packages/eslint-plugin/tests/rules/unbound-method.test.ts

```diff
@@ -590,5 +590,22 @@ class OtherClass extends BaseClass {
         },
       ],
     },
+    {
+      code: `
+const values = {
+  a() {},
+  b: () => {},
+};
+
+const { a, b } = values;
+      `,
+      errors: [
+        {
+          line: 7,
+          column: 9,
```

Comment:
Could you also add the `endColumn` here too? That way we know it's on the `a`, not `a, b` or some other odd range.